src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -0,0 +1,108 @@
+(ns quo2.components.colors.color-picker.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [reagent.core :as reagent]
+            [quo2.components.colors.color-picker.style :as style]))
+
+(def picker-colors
+  [{:name       "blue"
+    :color      "#2A4AF5"
```

Comment:
Please don't hard colors, use them from [colors file](https://github.com/status-im/status-mobile/blob/cbad7f4c87d3bfb555d2a4de3a5c51186d91ab3e/src/quo2/foundations/colors.cljs#L170) using function [custom-color-by-theme](https://github.com/status-im/status-mobile/blob/cbad7f4c87d3bfb555d2a4de3a5c51186d91ab3e/src/quo2/foundations/colors.cljs#L216).
(Looks like some colors are new, feel free to add them)

Reply:
Also, for the sake of consistency I would prefer naming keys as `:color-light` and `:color-dark`.

Reply:
Also we should add one test for each color checking the hex or rgb values, as this is a sensible component that will affect many others and we need to be sure we don't break color sets in the future.

Reply:
> Also, for the sake of consistency I would prefer naming keys as `:color-light` and `:color-dark`.

Once we use custom-color-by-theme function, we don't need to store these values.
`(custom-color-by-theme :blue 50 60)`

Reply:
> Also we should add one test for each color checking the hex or rgb values, as this is a sensible component that will affect many others and we need to be sure we don't break color sets in the future.

yes this sounds good

Reply:
If we are just going to compare values from foundations file, we probably need component test for that. We can just use `cljs.test`.

Reply:
> > Also, for the sake of consistency I would prefer naming keys as `:color-light` and `:color-dark`.
> 
> Once we use custom-color-by-theme function, we don't need to store these values. `(custom-color-by-theme :blue 50 60)`

Yes, if we can use just one key and use that function from the `quo2.foundations.colors` namespace, would be better

Reply:
@Parveshdhull In the new design, I noticed that color values are different compared to our existing customization colors. For example, the existing color code for blue 50 is `#4CB4EF`, while the new design uses `#2A4AF5`. Would it be acceptable to update the existing color codes with the new values?

Current blue 50:
<img width="70" alt="blue-50-old" src="https://user-images.githubusercontent.com/19279756/219564965-afb7f33b-58c8-42b6-9185-0fea7fc9563a.png">

New blue 50:
<img width="70" alt="blue-50-new" src="https://user-images.githubusercontent.com/19279756/219564989-d6ef9853-2c90-4286-a3ad-988aa0bf65b1.png">

Reply:
Yes please update them (Note: please check actual color value from inspect, label are wrong sometime)
![image](https://user-images.githubusercontent.com/17097240/219602658-c24da158-c115-475a-a937-1b2cc2cf0738.png)

Reply:
@Parveshdhull Could you please share the link to the design file with the color values? So that I can update all colors to the new values.

Reply:
https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?node-id=619%3A10064&t=dSqqKUVodgbr84nw-0